notebooks/Repo_Analysis.ipynb

```diff
@@ -99,58 +99,51 @@
      "output_type": "stream",
```

Comment:
<p>So based on my experience and your comment on open-services-group/metrics#30, it seems that if there's older data present then when fetching the new data sometimes there are inconsistencies. If that's the case, should we add a line to remove any existing data first? Like <code>rm -r srcopsmetrics/</code>?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-services-group/metrics/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Good idea! Updated the notebook to include this</p>

Reply:
thanks for the changes!